### DISCUSSION TITLE
Menu loading

### DISCUSSION THREAD

**Discussion by mgrfilipmarek** [rine77/homeassistantedupage#18] [Ideas]
It wold be nice to see menu for the week or month via HA.
Looks like its supported by API https://edupageapi.github.io/edupage-api/lunches.html

**Comment by rine77** [comment]
this could be a very nice thing. My school doesnt support it tho, so I need you to test it when I start developing ;)
/rine

**Reply by mgrfilipmarek** [reply]
Totally, I`m available for testing 😄

**Comment by rine77** [comment]
I created #33 to handle it
